superset-frontend/src/pages/ChartList/index.tsx

```diff
@@ -438,44 +434,30 @@ function ChartList(props: ChartListProps) {
       {
         Cell: ({
           row: {
-            original: { last_saved_by: lastSavedBy },
+            original: { owners = [] },
           },
-        }: any) => <>{changedByName(lastSavedBy)}</>,
-        Header: t('Modified by'),
-        accessor: 'last_saved_by.first_name',
+        }: any) => <FacePile users={owners} />,
+        Header: t('Owners'),
+        accessor: 'owners',
+        disableSortBy: true,
         size: 'xl',
       },
       {
         Cell: ({
           row: {
-            original: { last_saved_at: lastSavedAt },
+            original: { last_saved_at: changedOn, last_saved_by: changedBy },
           },
         }: any) => (
-          <span className="no-wrap">
-            {lastSavedAt ? moment.utc(lastSavedAt).fromNow() : null}
-          </span>
+          <AuditInfo
+            type={AuditInfoType.Modified}
+            date={changedOn}
```

Comment:
Shouldn't we use `changed_on_delta_humanized` here? I'm seeing an inconsistency between ChartList and for example DashboardList

<img width="1695" alt="image" src="https://github.com/apache/superset/assets/15073128/25d8eb9d-a96b-4a40-b0c8-8205a9f8d9dc">

<img width="1728" alt="image" src="https://github.com/apache/superset/assets/15073128/eed473f7-99c4-4b9d-a985-64459c122d9a">

Reply:
Nevermind, it was noted by Michael already 🙂